spec/rails_helper.rb

```diff
@@ -37,6 +37,14 @@ def common_org
     @common_org ||= Organization.find_or_create_by!(name: "Common Test Organization")
   end
 
+  def with_reporting_time_zone(&blk)
+    Time.use_zone(Period::REPORTING_TIME_ZONE) do
+      Groupdate.time_zone = Period::REPORTING_TIME_ZONE
+      blk.call
+      Groupdate.time_zone = nil
+    end
+  end
```

Comment:
Do we really want this to be available outside of tests for reporting? If we need it, I feel like that reflects some lack of separation between transactional and reporting models.